src/inc/schedule-output-functions.php

```diff
@@ -263,6 +263,58 @@ function wpcs_schedule( $atts, $content ) {
 	return $return;
 }
 
+/**
+ * Get tags for a session.
+ *
+ * @param int    $talk_id Post ID for session.
+ * @param string $tag_heading_txt Heading text for tags.
+ *
+ * @return string
+ */
+function wpad_get_tags_html( $talk_id ) {
+	$all_tags           = wp_get_post_terms( $talk_id, 'wpcs_session_tag' );
+	$filtered_tags_html = '';
+
+	if ( ! is_wp_error( $all_tags ) && ! empty( $all_tags ) ) {
+
+		// Filter out tags that appear in more than one post.
+		$filtered_tags = array_filter( $all_tags, function( $tag ) {
+			return $tag->count > 1;
+		} );
+
+		if ( ! empty( $filtered_tags ) ) {
+			foreach ( $filtered_tags as $tag ) {
+				$filtered_tags_html .= '<span class="talk-tag-wrapper">';
```

Comment:
Let's use `ul` and `li` for the tags semantics, so that we have some meaningful structure in the code.